### PR TITLE
One-on-one help on Gitcoin

### PR DESCRIPTION
**Pull request by zoek1**
##### Description

Add a new page in the Gitcoin website that embeds a video call and uses the Sablier protocol for the real-time finance part. The mentee streams money to the mentor while they are on the call.

Team members: @lcoenen @zoek1
##### Refers/Fixes

#5727 

##### Testing
1. Create a session
2. invite to one user sharing the url link or wait to someone join the session from the mentor panel
  - The mentee will be requested to deposit the DAI, this takes a while due to waiting the tx confirmation
3. A video session will be started from there screen, the mentee could stop the stream, the mentor could withdraw the funds.
4. Once the stream stopped, the session is finished and the users will be available to join or create other session.

See demo: https://youtu.be/IxD9zHaqPJs
[IMAGE ALT TEXT HERE](https://www.youtube.com/watch?v=IxD9zHaqPJs)

### CODE REVIEW

**Comment by danlipert** [comment]
@frankchen07 @PixelantDesign have y'all seen this?

**Comment by frankchen07** [comment]
@zoek1 & @lcoenen - this is really dope, I'm impressed!

@danlipert this is very interesting, and streaming money was definitely one of the ideas that was thrown around in several email threads

our user feedback about mentorship has been all over the board, but this I can see as a GREAT experiment. 

let's talk about what kind of reviews need to be done for internal hack week on this, if this is something @danlipert and team is interested in?

**Comment by danlipert** [comment]
Hey @zoek1 - can you rebase this and squash the migrations down so its one migration per app per PR? Then we'll merge it into master and get it deployed soon - thanks!

**Comment by zoek1** [comment]
Sure @danlipert!

File: app/dashboard/templates/profiles/header_details.html.bkp [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    <i class="fas fa-share-square"></i>
+  </a>
+  {% if verification %}
+    <button class="btn btn-sm animate-verify" data-container="body" data-toggle="popover" data-html="true" data-placement="bottom" data-trigger="hover click" data-content='
+    <p class="h6 my-2 text-left">Gitcoin Verified <img width="18" src="{% static "v2/images/badge-verify.svg" %}"></p>
+    <p>This contributor has completed the Gitcoin verification process, which means that the Gitcoin Core trusts their work.</p>
+    <p>Please note - use your judgement on whether this is the right contributor for your task.</p>
+    '><img src="{% static 'v2/images/badge-verify.svg' %}" alt=""></button>
+  {% endif %}
+<<<<<<< HEAD
```
Comment (octavioamu) [inline app/dashboard/templates/profiles/header_details.html.bkp]: something missing here

**Comment by octavioamu** [comment]
Guys how is  this going?

**Comment by zoek1** [comment]
Let me fix this in these days, i'll back to you ASAP.

**Comment by thelostone-mc** [comment]
@zoek1 you still getting this PR ready / you waiting on just a review ?

**Comment by thelostone-mc** [comment]
switching it to draft ! let me know @zoek1 when this is ready for review

**Comment by thelostone-mc** [comment]
Closing this out but will reopen if priorities change!

**Pull request closed without merging**